scripts/tests/test_validate_min_versions_in_sync.py

```diff
@@ -0,0 +1,83 @@
+import pathlib
+import sys
+
+import pytest
+import yaml
+
+if sys.version_info >= (3, 11):
+    import tomllib
+else:
+    import tomli as tomllib
+
+from scripts.validate_min_versions_in_sync import (
+    get_toml_map_from,
+    get_yaml_map_from,
+    pin_min_versions_to_yaml_file,
+)
+
+
+@pytest.mark.parametrize(
+    "src_toml, src_yaml, expected_yaml",
+    [
+        (  # random
```

Comment:
Instead of these comments could you put this comment in the file name like `random_deps.toml`

Also for the directory naming could you use something like `scripts/tests/data/`?